bamoe-ddl/postgresql/data-index.sql

```diff
@@ -125,7 +125,8 @@ CREATE TABLE nodes (
     name character varying(255),
     node_id character varying(255),
     type character varying(255),
-    process_instance_id character varying(255) NOT NULL
+    process_instance_id character varying(255) NOT NULL,
+    sla_due_date timestamp without time zone

```

Comment:
@yesamer I'm not sure about the `without time zone`. Is it added by pgsql?

Reply:
@pefernan Yes, exactly like all the attributes with timezone